server/views/app.ejs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
see comment regarding use of non-react view engine, in fact do we still need these?